### PR TITLE
Nebula config update

### PR DESCRIPTION
**Pull request by tagoylo**
AR bootfile checker was failing. Updated the nebula config generator command to add devices-role and status.

### CODE REVIEW

File: src/sdg/Gauntlet.groovy [hunk 1]
```diff
@@ -110,6 +110,8 @@ private def update_agent() {
                                     + ' --netbox-port=' + gauntEnv.netbox_port
                                     + ' --netbox-baseurl=' + gauntEnv.netbox_base_url
                                     + ' --netbox-token=' + gauntEnv.netbox_token
+                                    + ' --devices-status=' + gauntEnv.netbox_devices_status
```
Comment (kimpaller) [inline src/sdg/Gauntlet.groovy:113]: can you make devices to singular? devices->device
and kindly update any reference include gauntEnv.netbox_devices_tag 😅

File: vars/getGauntEnv.groovy [hunk 2]
```diff
@@ -32,10 +32,13 @@ private def call(hdlBranch, linuxBranch, bootPartitionBranch,firmwareVersion, bo
             docker_host_mode: true,
             update_nebula_config: true,
             nebula_config_source: 'netbox',
+            netbox_test_agent: false,
```
Comment (kimpaller) [inline vars/getGauntEnv.groovy:35]: What's netbox_test_agent variable for?

File: src/sdg/Gauntlet.groovy (hunk 1 above)
Comment (tagoylo) [reply src/sdg/Gauntlet.groovy]: [55aa74e](https://github.com/sdgtt/jenkins-shared-library/pull/77/commits/55aa74e1e3d75528c0b92d96c286262e058e6f92)

File: vars/getGauntEnv.groovy (hunk 2 above)
Comment (tagoylo) [reply vars/getGauntEnv.groovy:35]: netbox_test_agent is for when the nebula config to generate is of a test_agent.

**Review by kimpaller — APPROVED** [review]

**Pull request merged**